services/ui/ws/window_manager_display_root.cc

```diff
@@ -25,9 +25,24 @@ WindowManagerDisplayRoot::WindowManagerDisplayRoot(Display* display)
   properties[mojom::WindowManager::kName_Property] =
       std::vector<uint8_t>(name.begin(), name.end());
 
-  root_.reset(window_server()->CreateServerWindow(
-      window_server()->display_manager()->GetAndAdvanceNextRootId(),
-      properties));
+  // We pretend the WindowManagerDisplayRoot::root_ is created by the root
```

Comment:
maybe move this part to another fixup! c++ / mojo changes for 'external window mode' ?
I think it belongs to that part rather than deletion of windows.